components/emoji_picker/emoji_picker_tabs.jsx

```diff
@@ -74,9 +76,12 @@ export default class EmojiPickerTabs extends PureComponent {
                 pickerStyle = {...this.props.style};
             }
 
-            if (pickerStyle.top) {
+            if (pickerStyle.top || this.props.topOffset) {
```

Comment:
this should be `if (pickerStyle.top && this.props.topOffset) {` since it will break if `pickerStyle.top` is `undefined`